### ISSUE TITLE
[Bug]: Error while fetching via RSS: Error: connect ETIMEDOUT 172.217.169.46:443

### ISSUE THREAD

**Issue by Elastic-Mushroom** [FreeTubeApp/FreeTube#2461]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

When refreshing subscriptions or initially loading them via RSS the progress bar will go to ~95% (Picture attached) in under a second then freeze there for over 20 seconds before showing "Error: connect ETIMEDOUT 172.217.169.46:443" 

![freetube freeze2](https://user-images.githubusercontent.com/110774289/183295768-9831d118-3bbe-457e-9270-5a8c77fbde06.PNG)

### Expected Behavior

Subscriptions to update without error and at a similar speed to before the error started appearing 

### Issue Labels

API issue

### FreeTube Version

v0.17.0 

### Operating System Version

Windows 10 version 21H1 19044.1826

### Installation Method

Chocolatey

### Primary API used

Local API

### Additional Information

Encountered after updating to v0.17.0 via chocolatey but downloading a win portable v0.16.0 also has the same error.

Subscriptions do appear to be fetch correctly.

It appears to vary with different profiles and their subscriptions: Profile with 1 subscription never seems to error one with 18 will sometimes load fine usually on initial load but will display the error after some refreshes and Profile with 170 will always display the error four times when loaded or refreshed.

Changing Invidious Instance doesn't help and displays the same IP

This happens if RSS is chosen or forced (through too many subscriptions) but 

I put API issuse as one was required but I have no idea whether it is or what I should have put instead

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by Elastic-Mushroom** [comment]
Update I had this bug yesterday and earlier today but It is working normally now

**Comment by absidue** [comment]
Could you please export your subscriptions so we can reproduce the issue and also for testing fixes?

Settings -> Data Settings -> Export Subscriptions -> Export FreeTube (.db)

**Comment by Elastic-Mushroom** [comment]
The issue stopped and hasn't reoccurred in a week now.

Sorry for leaving it hanging. When I saw your comment I was still having the issue but wasn't super comfortable sending that (my profile had my name etc) so after exporting it I loaded it up in a virtual machine and simplified it to one profile with 150 which was still exhibiting the issue but when I was double checking everything the issue stopped in the virtual machine and my unmodified FreeTube. As the issue never re-occurred I didn't think there was much point sending the file.
